docs/pages/debugging/devtools-plugins.mdx

```diff
@@ -0,0 +1,201 @@
+
+To use the plugin, first install the package:
+
+<Terminal cmd={['$ npx expo install @dev-plugins/react-navigation']} />
+
+Then pass navigation root to the plugin in your app's entry point when running in development mode:
+
+<Tabs tabs={["Expo Router", "React Navigation"]}>
+  <Tab>
+  ```jsx app/_layout.js
+  import { useEffect, useRef } from 'react';
+  import { useRootNavigation, Slot } from 'expo-router';
+  import { useReactNavigationDevTools } from '@dev-plugins/react-navigation';
+
+  export default Layout() {
+    if (__DEV__) {
+      const navigationRef = useRef(null);
+      const rootNavigation = useRootNavigation();
+
+      useReactNavigationDevTools(navigationRef);
+
+      useEffect(() => {
+        navigationRef.current = rootNavigation;
+      }, [navigationRef.current]);
+    }
+
+    return <Slot />;
+  }
+
+  ```
+  </Tab>
+  <Tab>
+  ```jsx App.js
+  import { NavigationContainer, useNavigationContainerRef } from '@react-navigation/native';
+  import { useReactNavigationDevTools } from '@dev-plugins/react-navigation';
+
+  export default function App() {
+    const navigationRef = useNavigationContainerRef();
+
+    useReactNavigationDevTools(navigationRef);
+
+    return (
+      <NavigationContainer ref={navigationRef}>{/* ... */}</NavigationContainer>
+    );
+  }
+  ```
+  </Tab>
+</Tabs>
+
+Run `npx expo start`, press <kbd>shift</kbd> + <kbd>m</kbd> to open the list of dev tools, and then select the React Navigation plugin. This will open the plugin's web interface, which will show your navigation history as you navigate through your app.
+
+### Apollo Client
+
+Inspired by [react-native-apollo-devtools](https://github.com/razorpay/react-native-apollo-devtools), Apollo Client dev tools plugin lets you example cache, queries and mutations for the Apollo Client.
+
+To use the plugin, first install the package:
+
+<Terminal cmd={['$ npx expo install @dev-plugins/apollo-client']} />
+
+Then pass your client instance to the plugin in your app's root component, or wherever you wrap the rest of your app in the `ApolloProvider`:
+
+```jsx App.js
+import { ApolloProvider, ApolloClient, InMemoryCache } from '@apollo/client';
+import { useApolloClientDevTools } from '@dev-plugins/apollo-client';
+
+const client = new ApolloClient({
+  uri: 'https://demo.test.com/',
+  cache: new InMemoryCache(),
+});
+
+export default function App() {
+  if (__DEV__) {
+    useApolloClientDevTools(client);
+  }
+
+  return (
+    <ApolloProvider>
+      {/* ... */}
+    </ApolloProvider>
+  )
+}
+```
+
+Run `npx expo start`, press <kbd>shift</kbd> + <kbd>m</kbd> to open the list of dev tools, and then select the Apollo Client plugin. This will open the plugin's web interface, which will show your query history, cache, and mutations as your app performs Apollo Client operations.
```

Comment:
```suggestion
In the terminal, run `npx expo start`, press <kbd>shift</kbd> + <kbd>m</kbd> to open the list of dev tools, and then select the Apollo Client plugin. This will open the plugin's web interface, showing your query history, cache, and mutations as your app performs Apollo Client operations.
```